pkg/healthcheck/healthcheck_test.go

```diff
@@ -2422,7 +2422,7 @@ spec:
       serviceAccountName: linkerd-cni
       containers:
       - name: install-cni
-        image: cr.l5d.io/linkerd/cni-plugin:git-b4266c93
+        image: cr.l5d.io/linkerd/cni-plugin:v1.0.0
```

Comment:
Strangely, this tag was from a commit in mid-2020 so I bumped it manually to v1.0.0

Reply:
Great spot!